inc/namespace.php

```diff
@@ -267,6 +271,22 @@ function add_login_logo() {
 	<?php
 }
 
+/**
+ * Undocumented function
```

Comment:
😛

Reply:
🙈  It's one of those at the time of filling it out I started on the parameters and would "come back" to the last change!

FWIW I found these also in a search, not sure if it's worth going through and changing them too? ( obviously one if mine that's been changed now )
![Screenshot 2022-01-04 at 08 31 13](https://user-images.githubusercontent.com/16571365/148018746-d45e2de9-bf21-4c8d-9211-c181cafb16bf.png)

Reply:
Heh, classic! Can fix the cloud module one but the others aren’t too important.